src/site/constants/drupals.js

```diff
@@ -37,8 +37,9 @@ const ENABLED_ENVIRONMENTS = new Set([
 ]);
 
 const PREFIXED_ENVIRONMENTS = new Set([
-  ENVIRONMENTS.LOCALHOST,
-  ENVIRONMENTS.VAGOVSTAGING,
+  // ENVIRONMENTS.LOCALHOST,
```

Comment:
I assume we can remove this for now?

Reply:
The only issue with localhost is that Drupal Staging provides all of the hub landing pages, and content is still editing some them. I think this is still the right thing to do, but that's something we'll have to deal with.

Added this ticket to help with handling Heroku, https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17537.